.pyup.yml

```diff
@@ -0,0 +1,63 @@
+# configure updates globally
+# default: all
+# allowed: all, insecure, False
+update: all
+
+# configure dependency pinning globally
+# default: True
+# allowed: True, False
+pin: True
+
+# set the default branch
+# default: empty, the default branch on GitHub
+branch: dev
+
+# update schedule
+# default: empty
+# allowed: "every day", "every week", ..
+schedule: "every day"
+
+# search for requirement files
+# default: True
+# allowed: True, False
+search: True
+
+# Specify requirement files by hand, default is empty
+# default: empty
+# allowed: list
+requirements:
+  - requirements/staging.txt:
+      # update all dependencies and pin them
+      update: all
+      pin: True
+  - requirements/dev.txt:
+      # don't update dependencies, use global 'pin' default
+      update: False
+  - requirements/prod.txt:
+      # update insecure only, pin all
+      update: insecure
+      pin: True
+
+# add a label to pull requests, default is not set
+# requires private repo permissions, even on public repos
+# default: empty
+label_prs: update
+
+# assign users to pull requests, default is not set
+# requires private repo permissions, even on public repos
+# default: empty
+assignees:
+ - carl
+ - carlsen
+
+# configure the branch prefix the bot is using
+# default: pyup-
+branch_prefix: pyup/
+
+# set a global prefix for PRs
+# default: empty
+pr_prefix: "Bug #12345"
```

Comment:
IMHO it makes no-sense to have all PRs opened by pyup.io named `Bug #12345 […]`. I would not use this setting.